### ISSUE TITLE
Class numbers

### ISSUE THREAD

**Issue by xafarranxera** [sanwong15/yolo-v3-with-bdd#2]
Does your code assure that the same class numbers are assigned to the same classes in train and val label files?

Thanks.

**Comment by xafarranxera** [comment]
No, it doesn't. It is disastrous. If you don't persist these class numbers, then they are different for train and validation subsets, so DIFFERENT INDEXES ARE ASSIGNED FOR EACH CLASS IN TRAIN AND VALIDATION SUBSETS.

Besides from that, THE CENTER WIDHT AND CENTER HEIGHT COLUMNS IN EACH .txt ARE SWAPPED.

All of this causes the training process to fail completely. PLEASE TRY YOUR CODE BEFORE UPLOADING IT.

**Comment by SK124** [comment]
Hi @xafarranxera , did you implement yolov3 on bdd100k dataset?
can I get your source code ,if your implementation was successful
Thanks
